test/dd/test_state_preparation.cpp

```diff
@@ -0,0 +1,56 @@
+#include "dd/Package.hpp"
+
+#include <gtest/gtest.h>
+
+class StatePreparation : public testing::TestWithParam<qc::Qubit> {};
+
+extern std::vector<std::string> generateWStateStrings(const std::size_t length);
```

Comment:
```suggestion
extern std::vector<std::string> generateWStateStrings(std::size_t length);
```
I guess this is the only warning left to fix here.